pod.go

```diff
@@ -699,11 +699,6 @@ func (p *Pod) startSetStates() error {
 		return err
 	}
 
-	err = p.setContainersState(podState.State)
```

Comment:
If we removing the container state setting, the function needs to be renamed to `startSetState()`.

Reply:
renamed